includes/corrections/class-corrections.php

```diff
@@ -171,7 +171,7 @@ public static function register_post_type() {
 		\register_post_type( self::POST_TYPE, $args );
 
 		$rewrite_rules_updated_option_name = 'newspack_corrections_rewrite_rules_updated';
-		if ( get_option( $rewrite_rules_updated_option_name ) !== true ) {
+		if ( get_option( $rewrite_rules_updated_option_name ) !== '1' ) {
```

Comment:
WDYT checking for `false === ...` ?

Reply:
Alright! Done in b39599577306463e89d536a09bdcde8a7b610699